packages/menu/src/MenuItem.tsx

```diff
@@ -165,6 +165,12 @@ const MenuItem: ExtendableBox<
         ),
       });
 
+    const textStyles = cx(titleTextStyle, {
```

Comment:
What's the value in separating this variable out, rather than incorporating it into the existing `cx()`? We even already have handling for active, disabled, and focus there that we could merge this with.